aries-site/src/pages/templates/forms.mdx

```diff
@@ -250,3 +258,40 @@ In cases where the user is given an opportunity to express themselves or would l
   href="#sign-in"
   alt="HPE Design System Form where required and optional can be skipped"
 />
+
+## Full-page form
+
+- Use full-page forms for create and edit flows, irrespective of the size of the form.
+- Use a single-column full-page form instead of multi-column layouts.
+- Full-page forms shoud be center-aligned.
```

Comment:
Can we see this? How does it work within Page, PageContent, and in relation to PageHeader?

Reply:
yeah so I have this example locally but was waiting on mutli-select but I did wrap it in `page` and use `pageHeader` and `pageContent`